autofocus/build_dataset/lpz_2016_2017/ops.py

```diff
@@ -42,11 +42,13 @@ def record_mean_brightness(
     image: np.array, inpath: PathOrStr, log_dict: DefaultDict[str, dict]
 ) -> np.array:
     """
-    Record whether image is grayscale.
+    Record the mean brightness of image.
 
     In this dataset, grayscale images have been saved as three-channel
```

Comment:
We can delete this note about grayscale images, which isn't really relevant to this function. In fact, I think we can just delete this paragraph of the docstring.